crashlogger.js

```diff
@@ -23,9 +23,9 @@ let hadException = false;
  * Logs when a crash happens to console, then e-mails those who are configured
  * to receive them.
  *
- * @param {Error} err
+ * @param {NodeJS.ErrnoException} err
```

Comment:
It is if anything accesses properties of the error apart from its stack before passing it to the crashlogger, since the type of those errors won't match `Error`